### ISSUE TITLE
Enum querying using "or" condition

### ISSUE THREAD

**Issue by hgani** [lwe/simple_enum#106]
Hi,

Sorry if I've missed it but I haven't found any definitive guide for easily querying using "or" condition.

Let say we have:

  as_enum :status, { active: 0, pending: 1, declined: 11 }

And we want to query records that are either active or pending. It doesn't seem that there's a shortcut for doing this yet. I think this can be implemented as part of the solution suggested in https://github.com/lwe/simple_enum/issues/95

So, we can do something like the following:

  Item.where_with_enum(status: [:active, :pending])

Thanks

**Comment by lwe** [comment]
Good point, it indeed makes sense to support this and I think we would get support for this out of the box.

**Comment by aledalgrande** [comment]
I think this one can be associated to #95 and closed.
